target/linux/x86/base-files/lib/preinit/01_sysinfo

```diff
@@ -21,6 +21,10 @@ do_sysinfo_x86() {
 		"Supermicro:Super Server")
 			continue
 			;;
+		"PC Engines:APU")
+			product="apu1"
```

Comment:
Could you please clarify this for me? Why does only the apu1 require an extra entry here?

Reply:
Hello aparcar!
@zokl explained this very well in his post:

> APU boards are usually delivered with FW <4.0.8. This is a very old FW with a lot of erroneous information in the DMI, typically the type of board or its name. Newer versions fix this (https://pcengines.github.io).
...
The current version of leds-apu.ko used in the kernel does not take into account the state of the DMI before and after FW version 4.0.8 (needs to be modified).

The issue with the older firmware versions only applies to the APU board revision/version 1.

Reply:
https://github.com/openwrt/openwrt/pull/3135#issuecomment-748667177